Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -410,12 +421,17 @@ final class DeclarationTests: XCTestCase {
 
     assertParse(
       """
-      private(1️⃣get, set2️⃣, didSet) var a = 0
+      private(1️⃣get, set, didSet) var a = 0
       """,
       diagnostics: [
-        DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected code 'get,' in modifier"),
-        DiagnosticSpec(locationMarker: "2️⃣", message: "unexpected code ', didSet' in modifier"),
-      ]
+        DiagnosticSpec(
+          message: "expected 'set' in modifier",
+          fixIts: ["replace 'get, , didSet' with 'set'"]
+        )
+      ],
+      fixedSource: """
+        private(set) var a = 0
+        """
     )
 
     assertParse(
```

Comment:
This assert and fixedSource is not correct either. 
But that is in the parser and not the diagnostic generator. 

Think that is a PR for it self